<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emoji</class>
 <widget class="QWidget" name="emoji">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_28">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_31">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_38">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_35">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_34">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_36">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_37">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_39">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>67</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>33</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_40">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_41">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-5</y>
     <width>350</width>
     <height>160</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_41</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_1</zorder>
  <zorder>pushButton_21</zorder>
  <zorder>pushButton_24</zorder>
  <zorder>pushButton_28</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>pushButton_18</zorder>
  <zorder>pushButton_22</zorder>
  <zorder>pushButton_27</zorder>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_16</zorder>
  <zorder>pushButton_23</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>pushButton_17</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>pushButton_25</zorder>
  <zorder>pushButton_26</zorder>
  <zorder>pushButton_31</zorder>
  <zorder>pushButton_38</zorder>
  <zorder>pushButton_32</zorder>
  <zorder>pushButton_33</zorder>
  <zorder>pushButton_35</zorder>
  <zorder>pushButton_34</zorder>
  <zorder>pushButton_36</zorder>
  <zorder>pushButton_37</zorder>
  <zorder>pushButton_39</zorder>
  <zorder>pushButton_20</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_30</zorder>
  <zorder>pushButton_29</zorder>
  <zorder>pushButton_19</zorder>
  <zorder>pushButton_40</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
